### PR TITLE
fix(ext/io): Use CreateNamedPipeW for windows pipes

### PR DESCRIPTION
**Pull request by mmastrac**

### CODE REVIEW

File: ext/io/winpipe.rs [hunk 1]
```diff
@@ -43,24 +42,21 @@ pub fn create_named_pipe() -> io::Result<(RawHandle, RawHandle)> {
 fn create_named_pipe_inner() -> io::Result<(RawHandle, RawHandle)> {
   static NEXT_ID: AtomicU32 = AtomicU32::new(0);
   // Create an extremely-likely-unique pipe name from randomness, identity and a serial counter.
-  let pipe_name = format!(
-    r#"\\.\pipe\deno_pipe_{:x}.{:x}.{:x}\0"#,
+  let pipe_name_utf8 = format!(
+    r#"\\.\pipe\deno_pipe_{:x}.{:x}.{:x}"#,
```
Comment (littledivy) [inline ext/io/winpipe.rs:46]: Do we handle name collisions?

Comment (mmastrac) [reply ext/io/winpipe.rs:46]: Via retry, unfortunately the only way to do this on Windows

**Review by littledivy — APPROVED** [review]
LGTM, can you add a small PR description?

**Comment by mmastrac** [comment]
> LGTM, can you add a small PR description?

I am not sure I'm going to land this one -- I have a sneaking suspicion this is not the problem 😢

**Pull request closed without merging**